src/main/java/uk/gov/mca/beacons/service/model/BeaconPerson.java

```diff
@@ -27,6 +27,8 @@
 
   private String telephone;
 
+  private String alternativeTelephoneNumber;
```

Comment:
Added missing columns.

Errors thrown up during integration tests.  These had been missed when adding the first migration script to align the database tables with the models within the frontend.